app/assets/stylesheets/main.scss

```diff
@@ -14,4 +14,21 @@
 }
 .footer-content {
   padding: 10px 10px;
-}
\ No newline at end of file
+}
+
+.profilebox {
+  width: 500px;
+  text-align: center;
+}
+
+.profilebox label {
+  text-align: left;
+}
+
+.profilebox label.required abbr {
+  color: red;

```

Comment:
move to bulma_ext or something ... required abbr must be globally defined for all forms